mkt/reviewers/views.py

```diff
@@ -1005,6 +1006,27 @@ def post(self, request, pk, region, *args, **kwargs):
         return Response({'approved': bool(form.cleaned_data['approve'])})
 
 
+class _WebsiteAction(object):
+    permission_classes = [GroupPermission('Apps', 'Review')]
```

Comment:
Can you add a new `Websites:Review` permission, and change it here and in `ReviewersSearchView` ? 

@eviljeff do you know which groups we should assign the permission to ? Do we need a new group ? (Note that this isn't necessary for MOW Phase 0, only Phase 1, so we can fix things later, there is no rush)